### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by devartes**

### CODE REVIEW

**Review by mcandeia — CHANGES_REQUESTED** [review]
Looks good!

I want to add one more thing: Ahead-of-time builds.

To do that, you need to:

1. Change dev.ts to include fresh.config.ts (I think you can copy from here: https://github.com/deco-sites/storefront/blob/main/fresh.config.ts), please remind that you need to change both main.ts and dev.ts to import from fresh.config.ts. Also: Pay attention on this line: https://github.com/deco-sites/storefront/blob/main/dev.ts#L10
2. Change from import_map.json to deno.json, as you can see [here](https://github.com/deco-sites/storefront/blob/main/deno.json#L2), import maps and deno.json got merged. 
3. Use the same fresh version as storefront uses
4. Add a deco-deploy.yaml workflow exactly like this one => https://github.com/deco-sites/storefront/blob/main/.github/workflows/deco-deploy.yaml. The checks will start to fail, don't worry, this is expected.

File: apps/site.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return {
+    state,
+    manifest,
+    dependencies: [
+      website(state),
+    ],
+  };
+}
+
+export type Storefront = ReturnType<typeof Site>;
```
Comment (mcandeia) [inline apps/site.ts:21]:
```suggestion
export type SiteApp = ReturnType<typeof Site>;
```

File: apps/site.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+export type AppContext = AC<Storefront>;
```
Comment (mcandeia) [inline apps/site.ts:22]:
```suggestion
export type AppContext = AC<SiteApp>;
```

File: apps/site.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+export { onBeforeResolveProps } from "apps/compat/$live/mod.ts";
```
Comment (mcandeia) [inline apps/site.ts:23]:
```suggestion
export { onBeforeResolveProps } from "apps/website/mod.ts";
```

File: import_map.json [hunk 4]
```diff
@@ -1,15 +1,17 @@
 {
   "imports": {
     "deco-sites/start/": "./",
-    "deco-sites/std/": "https://denopkg.com/deco-sites/std@1.12.19/",
-    "$live/": "https://denopkg.com/deco-cx/live@1.29.9/",
-    "$fresh/": "https://denopkg.com/deco-cx/fresh@1.3.0/",
+    "deco-sites/std/": "https://denopkg.com/deco-sites/std@1.22.11/",
+    "$live/": "https://denopkg.com/deco-cx/deco@1.44.1/",
```
Comment (mcandeia) [inline import_map.json:5]: remove $live occurrences

File: import_map.json [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
+    "$fresh/": "https://denopkg.com/deco-cx/fresh@1.4.4/",
     "preact": "https://esm.sh/preact@10.15.1",
     "preact/": "https://esm.sh/preact@10.15.1/",
```
Comment (mcandeia) [inline import_map.json:8]:
```suggestion
    "preact": "https://esm.sh/preact@10.16.1",
    "preact/": "https://esm.sh/preact@10.16.1/",
```

File: runtime.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import { forApp } from "$live/clients/withManifest.ts";
+import type { Storefront } from "./apps/site.ts";
+
+export const Runtime = forApp<Storefront>();
```
Comment (mcandeia) [inline runtime.ts:4]:
```suggestion
export const Runtime = forApp<SiteApp>();
```

File: runtime.ts [hunk 7]
```diff
@@ -0,0 +1,4 @@
+import { forApp } from "$live/clients/withManifest.ts";
+import type { Storefront } from "./apps/site.ts";
```
Comment (mcandeia) [inline runtime.ts:2]:
```suggestion
import type { SiteApp } from "./apps/site.ts";
```

**Review by mcandeia — CHANGES_REQUESTED** [review]

File: apps/site.ts (hunk 1 above)
Comment (devartes) [reply apps/site.ts]: ok
image

File: apps/site.ts (hunk 2 above)
Comment (devartes) [reply apps/site.ts]: ok
image

File: apps/site.ts (hunk 3 above)
Comment (devartes) [reply apps/site.ts]: ok
image

File: import_map.json (hunk 4 above)
Comment (devartes) [reply import_map.json]: import_map.json has been deleted
image

File: import_map.json (hunk 5 above)
Comment (devartes) [reply import_map.json]: import_map.json has been deleted
image

File: .release.json [hunk 8]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (mcandeia) [inline .release.json:1]: remove this

**Review by mcandeia — APPROVED** [review]
add the`.env` and remove .release.json and shipit!

File: .release.json (hunk 8 above)
Comment (devartes) [reply .release.json]: removed
image

**Pull request merged**